payment-handler/can-make-payment-event.https.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<!-- Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
+<meta charset="utf-8">
+<title>Tests for CanMakePaymentEvent</title>
+<link rel="help" href="https://w3c.github.io/payment-handler/#the-canmakepaymentevent">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+promise_test(async t => {
+  await navigator.serviceWorker.register('app-can-make-payment-false.js');
+  const registration = await navigator.serviceWorker.ready;
+  await registration.paymentManager.instruments.set('instrument-key', {
+    name: 'Test Payment Method',
+    enabledMethods: 'https://w3c-test.org',
+  });
+  await promise_rejects(t, 'NotSupportedError', (new PaymentRequest([{
```

Comment:
nit: please split this up a bit, otherwise, it's a bit unclear as to what might be causing the `NotSupportedError`:

```JS
const request = new PaymentRequest(...);
promise_reject(t, 'NotSupportedError', request.show())
```

Reply:
Split up.